plugin.xml

```diff
@@ -108,9 +108,7 @@
 				</intent-filter>
 			</receiver>
             <service android:enabled="true"
-                     android:exported="true"
```

Comment:
@dopry Thank you for your contribution! Is there any specific reason for dropping the export of the service? Is this a necessary change for the version upgrade?